chaoscenter/subscriber/pkg/events/workflow.go

```diff
@@ -67,15 +64,15 @@ func WorkflowEventWatcher(stopCh chan struct{}, stream chan types.WorkflowEvent,
 		informer = f.Argoproj().V1alpha1().Workflows().Informer()
 		// Start Event Watch
 	}
-	go startWatchWorkflow(stopCh, informer, stream, int64(startTime))
+	go ev.startWatchWorkflow(stopCh, informer, stream, int64(startTime))
```

Comment:
Have you ensured that only one go-routine has access to this struct? If not, you might have to use `atomic` types or a mutex to prevent race conditions

Reply:
Hi @DarthBenro008 , I'm not sure if I have ensured that there's only one go routine to access the struct.
How can I ensure/check it?

Thank you!